examples/forecasting/advanced_forecasting_example.py

```diff
@@ -0,0 +1,65 @@
+from autogluon.forecasting import ForecastingPredictor
+from autogluon.forecasting import TabularDataset
+import autogluon.core as ag
+
+train_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/train.csv")
+test_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/test.csv")
+static_features = TabularDataset("./COV19/toy_static_features.csv")
```

Comment:
Let's put the static features in s3, if you can send the file to me

Reply:
I've sent it to you on slack.